mlrun/model_monitoring/db/v3io_tsdb_reader.py

```diff
@@ -125,10 +179,59 @@ def read_data(
 
     for metric in metrics_without_data.values():
         metrics_values.append(
-            ModelEndpointMonitoringResultNoData(
+            ModelEndpointMonitoringMetricNoData(
                 full_name=metric.full_name,
                 type=ModelEndpointMonitoringMetricType.RESULT,
             )
         )
 
     return metrics_values
+
+
+def df_to_metrics_values(
```

Comment:
I don't want to delay this PR because of it but in general next time I suggest to add more description. All of the functions in this file are missing documentation. It's more about why and when each function is important and less about what it's doing. As a person who didn't wrote the code it takes me time to understand and explore why each function is needed.

Reply:
Ok. Improved a bit in 1ff9e6c.